### PR TITLE
add Header and Footer component

### PR DESCRIPTION
**Pull request by hsnice16**
### This PR will add code for the Header and Footer component

In this PR, converted header and footer from HTML to react code

**Don't go on the number of additions, it is showing like this because of package-lock.josn file**

### Please review and give your feedback

---
image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { createRoot } from "react-dom/client";
+
+import { BrowserRouter } from "react-router-dom";
+import App from "./App";
+
+const container = document.getElementById("root");
```
Comment (vrx29) [inline src/index.js:7]: great that you have used React 18 syntax.

**Review by vrx29 — APPROVED** [review]
Looks like just the basic setup for your app, you can merge it I didn't find any issues, everything is perfect.

**Review by pratik-kumar-sahu — APPROVED** [review]
- UI & Code looks clean.
- Nice use of external packages for text-style.
- Overall looks good.

**Comment by hsnice16** [comment]
Thank you for reviewing,

**Pull request merged**